index.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Landing Page</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+   <div id="header">
+     
+    <img src="images\original_trombones.png" alt="company-logo" id="header-img">
```

Comment:
- [ ] Nice job adding a logo. Don't you think it will be nicer if the logo acts as a link to the home page. This will ease access around the page. please do implement this.